scripts/ci/check_identity.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python
```

Comment:
we are going python3 only, this can be changed to be python3, the problem we will face soon and already having on some OSes is that python3 is installed as python, not python3, which will start causing issues also when distros start dropping python2/python3 naming and just install python3 as python...